src/prsht.rs

```diff
@@ -0,0 +1,309 @@
+// Copyright © 2015, skdltmxn
+// Licensed under the MIT License <LICENSE.md>
+//! Interface for the Windows Property Sheet Pages
+#[repr(C)] #[allow(missing_copy_implementations)]
+pub struct PSP {
+    unused: ::c_void,
+}
+pub type HPROPSHEETPAGE = *mut PSP;
+
+#[repr(C)] #[allow(missing_copy_implementations)]
+pub struct _PROPSHEETPAGEA {
```

Comment:
Could you instead have this be `PROPSHEETPAGEA` and just have it be a typedef to `PROPSHEETPAGEA_V4`?